ci/test.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
+php -v
```

Comment:
Did you mean to leave this here? It definitely doesn't hurt, but I wasn't sure if it was intended.

Reply:
Going to drop it, since I believe we can be _reasonably_ sure we're going to get the right version of PHP since we're using their image. If it was more of a black-box container, it might be more useful to have this somewhere in the log.